app-ios/tutanotaTests/AlarmManagerTest.swift

```diff
@@ -120,7 +120,7 @@ class AlarmManagerTest : XCTestCase {
         frequency: .daily,
         interval: 1,
         timeZone: dateProvider.timeZone.identifier,
-        endCondition: .count(times: 2)
+        endCondition: .count(times: 2), excludedDates: []
```

Comment:
formatting

Reply:
hopefully fixed xcode and webstorm do not get along